lib/model/visitor_exposed.dart

```diff
@@ -0,0 +1,21 @@
+// This class represent the Object returned when the activate is done with success.
+
+class VisitorExposed {
+  // visitorId
+  final String visitorId;
```

Comment:
I feel its redundant to have the `visitor` prefix for those fields, no ? The class is already explicit about the Visitor part

Reply:
Ahh yes ... , Since i named those fields with others I'll notice them to forward your suggestion . 👍